xChrome.psd1

```diff
@@ -12,7 +12,7 @@
 # RootModule = ''
 
 # Version number of this module.
-ModuleVersion = '1.1.0.0'
+ModuleVersion = '1.0.2.0'
```

Comment:
I am not incrementing the version number. I'm just updating it to match what is actually on the PS Gallery.